recipes/bcftools/1.5/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version="1.5" %}
+
+package:
+  name: bcftools
+  version: {{ version }}
+
+build:
+  number: 4
+
+source:
+  url: https://github.com/samtools/bcftools/releases/download/{{ version }}/bcftools-{{ version }}.tar.bz2
+  sha256: 9ff75a680891467776d7064f41deb6be7a595030ba807494727cd5bee7842301
+  patches:
+    - b37b03e815cd4f25af70a57c08449f8c719dcc6a.patch # from https://github.com/samtools/bcftools/pull/645
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+  host:
+    - openssl # [not osx]
+    - bzip2
+    - curl
+    - gsl
+    - xz
+    - zlib
+  run:
+    - openssl # [not osx]
+    - bzip2
+    - curl
+    - gsl
+    # Currently removed due to size and dependency issues
+    #- matplotlib # for plot-vcfstats
+    #- tectonic # for plot-vcfstats
+    - xz
+    - zlib
+    - perl
```

Comment:
We really need perl?!?

Reply:
No idea, just took it from the current recipe.

Reply:
It's needed by several scripts like vcfutils.pl , see https://github.com/bioconda/bioconda-recipes/pull/17179